### ISSUE TITLE
Cannot open studio

### ISSUE THREAD

**Issue by sree526** [Wakanda/wakanda-issues#174]
I have downloaded 64bit community version studio but when I open it and try to create a new solution its pops open new window and displays nothing. After that I tried to open a file from my system still it displays nothing. My system config is 64bit OS, 4 GB ram, i3 processor. I don't know whats' wrong. Do I need to configure anything for this? Help me out.
Thanks in advance

**Comment by JegoYann** [comment]
Hello @sree526 ,
Are you running Wakanda Studio inside a virtual machine ? The embed Chrome in Wakanda Studio is not working nicely in a virtual machine.
The best would be to use it in a non virtual machine.

Regards,
Yann

**Comment by sree526** [comment]
I am running it in my local machine only :-(. I am facing one more problem I can't create password while creating the user credentials in the directory .. Do you have any idea? Is that option available only for paid version?

**Comment by JegoYann** [comment]
User password is available with no restriction in the community version. The password inputs are in the column at the right of the directory. Can you share a small screencast to help me visualize your issue ?

**Comment by sree526** [comment]
![error](https://cloud.githubusercontent.com/assets/18196568/25173686/b5c41faa-2512-11e7-9e17-2ce3f0a96040.PNG)

Here you go... U can see there is no password entry option at the right of the directory

**Comment by JegoYann** [comment]
The password input is displayed above the `belongs to groups` section which is hidden.
So hide the console using the button on the top right to get more screen space.

![screen shot 2017-04-19 at 11 47 19](https://cloud.githubusercontent.com/assets/3337770/25173878/fdb74aa2-24f5-11e7-93e6-b09b602858fd.png)

**Comment by sree526** [comment]
Thank You... :-) .. but still the studio is not working in my laptop . I'll post the screenshot here

**Comment by sree526** [comment]
![error](https://cloud.githubusercontent.com/assets/18196568/25174165/2268c696-2514-11e7-84aa-dfbda4e44075.PNG)

**Comment by JegoYann** [comment]
Yep, looks like embedded chrome is not working on your configuration. Until now, this issue was only reproducible on a VM. We have to check it.

Can you describe your laptop ?
You say: 64bit OS, 4 GB ram, i3 processor
- Is it a Surface book ? Mac Book ?
- Which GPU ?
- Which Windows Version ?
- Screen size ?

**Comment by sree526** [comment]
Its windows 7, screensize 15 inches, laptop: Dell Latitude, GPU: Intel

**Comment by JegoYann** [comment]
Maybe you have an issue with Windows 7 ? Only Windows 10 is supported.

**Comment by sree526** [comment]
nope I'm using it in my friends laptop he has same configuration and its working fine in his system that's the reason I've asked does it require any depedencies?

**Comment by sree526** [comment]
![servererror](https://cloud.githubusercontent.com/assets/18196568/25175042/e466bb2a-2516-11e7-9989-44d33f0ed82a.PNG)
why do you get this error?

**Comment by JegoYann** [comment]
(1)
There is no dependencies i'm aware of. Do you have an open Chrome browser running aside with Wakanda Studio ? If so, try to start Wakanda Studio with no Chrome browser open.

(2)
Probably because another software is using the port 8081 or 8080 used by Wakanda Server.
Two solutions:
- easy: Stop the incriminated software
- advanced:
  - Change the Project port through settings interface (from 8081 to 8180 for example)
  - Change the Admin port through Server CLI parameters `--admin-port 8180`

**Comment by sree526** [comment]
1.created user credentials in directory and everything is working fine but everytime showing incorrect username or password error. even after entering correct credentials.
2. One more error I'm facing is in Angular component Property catalogue doesn't exist on type wakanda...

Do you have any idea about these errors??

**Comment by JegoYann** [comment]
(1)
- Is the password well saved ? Did you click on `done` button below the password input ? Did you type the password in both inputs ? Did you try with a simple password (like `aaa`).
- The authentification is active when one user is added to the `admin` group.
- Is the studio asking you to login ?

(2)
I'm not an expert in Angular component. I cannot advise you on this. Did you follow this tutorial ? https://wakanda.github.io/doc/#/guide?section=main-data-cs-instance

**Comment by sree526** [comment]
yup password is well saved. I have pressed done button . I have typed it in both inputs. I used password with just one letter.
Should I add the user to admin group to get authenticated? It's not studio that's asking to login . I have created an angular 4 app.
2. That's not the error from angular component. It's the error of wakanda service, there is no catalog in that service but in tutorial they have used it. I don't know what to do now.

**Comment by sree526** [comment]
I added users to admin group but still errors persist

**Comment by JegoYann** [comment]
Can you add screenshots to help me visualize your issue ?

**Comment by sree526** [comment]
![loginerror](https://cloud.githubusercontent.com/assets/18196568/25179456/a25b41c2-2527-11e7-8036-5babf03dc1a8.PNG)

**Comment by JegoYann** [comment]
Did you reload the server after you update the credential ? Either by clicking on `Run` button or `Reload` button.

**Comment by sree526** [comment]
Yup done that... still no luck :-(

**Comment by JegoYann** [comment]
Can you send me your solution to take a look ?

**Comment by hamzahik** [comment]
For the solution creation window, check that your documents/wakanda/extensions folder is empty.

On the screenshot you've shared there are errors on the tasks view. Do you still have those ? If you do remove the node_modules folder and click the run button again.

**Comment by sree526** [comment]
I made login work. But I can't make other CRUD operations to work. I'm building application in Angular4

**Comment by sree526** [comment]
newTodo() {
        this.wakanda.catalog.then(ds => {
            let todo = ds['Todo'].create({
                label: this.newTodoText,
                completed: false
            }); 
This is the code they have given in documentation for creating and saving an object to db. But its throwing an error catalog not found. Actually yes its not there in the wakanda.service but they have used it in tutorial. How to create that please help me out

**Comment by JegoYann** [comment]
Try with `this.wakanda.getCatalog().then` instead `this.wakanda.getCatalog.then` as written in the tutorial.

**Comment by JegoYann** [comment]
Try with `this.wakanda.getCatalog()` instead of `this.wakanda.catalog`.

**Comment by hamzahik** [comment]
https://wakanda.github.io/doc/#/guide?section=main-data-cs-instance

**Comment by hamzahik** [comment]
@sree526 are you still experiencing those issues ?
